features/networking/multus-ipv6.feature

```diff
@@ -50,3 +50,70 @@ Feature: Multus-CNI ipv6 related scenarios
       | curl | -g | -6 | --connect-timeout | 5 | [<%= cb.pod1_multus_ipv6 %>]:8080 |
     Then the step should succeed
     And the output should contain "Hello OpenShift"
+
+  # @author weliang@redhat.com
+  # @case_id OCP-38521
+  @admin
+  Scenario: IPv6 testing for OCP-31999: Whereabouts should exclude IPv6 ranges
+  # Bug https://bugzilla.redhat.com/show_bug.cgi?id=1913062
+  # Bug https://bugzilla.redhat.com/show_bug.cgi?id=1917984
+  # Make sure that the multus is enabled
+    Given the master version >= "4.6"
+    And the multus is enabled on the cluster
+    Given the default interface on nodes is stored in the :default_interface clipboard
+    And evaluation of `node.name` is stored in the :target_node clipboard
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    Given I obtain test data file "networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-excludeIP.yaml"
+    When I run oc create as admin over "whereabouts-excludeIP.yaml" replacing paths:
+      | ["metadata"]["name"]      | whereabouts-excludeipv6                                                                                                                                                                                                  |
+      | ["metadata"]["namespace"] | <%= project.name %>                                                                                                                                                                                                      |
+      | ["spec"]["config"]        | '{ "cniVersion": "0.3.0", "name": "whereabouts", "type": "macvlan", "mode": "bridge", "ipam": { "type": "whereabouts", "range": "fd00:dead:beef:1::1-fd00:dead:beef:1::4/64", "exclude": ["fd00:dead:beef:1::2/128"] } }'|
+    Then the step should succeed
+
+    # Create the three pods which consumes the custom resource
+    Given I obtain test data file "networking/multus-cni/Pods/IPv6/ipv6-pod.yaml"
+    When I run oc create over "ipv6-pod.yaml" replacing paths:
```

Comment:
@weliang1 I suggest an optional improvement here to assist debugging in future for people not familier with multus or trying to debug this flow. 
pod ```metadata.name``` and ```metadata.annotations.k8s.v1.cni.cncf.io/networks``` are not defined here because we are fetching their default values from ipv6-pod.yaml which is logically correct but it might be good to define them here (like what you did at Line 103 and 104) considering ipv6-pod.yaml is just a template. That way a debugger can correlate further steps without going and search for ipv6-pod.yaml file to look at. Doing this will depict that we are leveraging ```whereabouts-excludeipv6``` def created before this step and with any pod name. Just a small suggestion from readability and debugging perspective

Reply:
@anuragthehatter Agree, just fixed it. Thanks!